src/Utility/ExprFunctions.C

```diff
@@ -698,3 +699,16 @@ template class EvalMultiFunction<TensorFunc,Tensor,Real>;
 template class EvalMultiFunction<TensorFunc,Tensor,autodiff::var>;
 template class EvalMultiFunction<STensorFunc,SymmTensor,Real>;
 template class EvalMultiFunction<STensorFunc,SymmTensor,autodiff::var>;
+
+
+namespace utl {
+
+RealFunc* parseExprRealFunc(const std::string& function, bool autodiff)
```

Comment:
In such cases, I rather prefer the syntax
`RealFunc* utl::parseExprRealFunc(...)`
to reduce the need for the extra brace level. I guess they are equivalent in real.

Reply:
done.